index.html

```diff
@@ -285,6 +285,110 @@ <h2>
         identifier=]. See also section [[[#protocol-registry]]].
       </dd>
     </dl>
+    <h3>
+      Credential Request Coordinator
+    </h3>
+    <p>
+      The <dfn>credential request coordinator</dfn> is a user-agent-defined
+      component responsible for mediating [=digital credential=] interactions
+      within a given [=top-level traversable=].
+    </p>
+    <p>
+      Each [=top-level traversable=] has a single associated [=credential
+      request coordinator=].
+    </p>
+    <p>
+      The [=credential request coordinator=]:
+    </p>
+    <ul>
+      <li>Ensures that only one [=digital credential=] interaction is active at
+      any time. This includes via both the:
+        <ul>
+          <li>{{DigitalCredential/[[DiscoverFromExternalSource]](origin,
+          options, sameOriginWithAncestors)}} internal method.
+          </li>
+          <li>{{DigitalCredential/[[Create]](origin, options,
+          sameOriginWithAncestors)}} internal method.
+          </li>
+        </ul>
+      </li>
+      <li>Handles state transitions across the following <dfn data-for=
+      "credential request coordinator" data-local-lt="state">interaction
+      states</dfn>:
+        <ul>
+          <li>
+            <dfn data-dfn-for="credential request coordinator">Idle</dfn>: No
+            credential interaction is currently in progress.
+          </li>
+          <li>
+            <dfn data-dfn-for=
+            "credential request coordinator">Presenting</dfn>: A credential
```

Comment:
`Presenting` is very tailored for presentation, but doesn't work well with Issuance.

```suggestion
            "credential request coordinator">Ongoing</dfn>: A credential
```

Reply:
Good call